rds-pgbadger.rb

```diff
@@ -7,18 +7,28 @@
 require 'aws-sdk-core'
 
 options = {}
+options[:logs_directory] = "logs"
+options[:reports_directory] = "reports"
+options[:parallel] = true
+#options[:incremental] = false
```

Comment:
Is this something you'd like to add in the future? Otherwise I'd say it should be removed for now.